docs/make.jl

```diff
@@ -71,16 +72,28 @@ makedocs(;
     repo = Remotes.GitHub("qutip", "QuantumToolbox.jl"),
     sitename = "QuantumToolbox.jl",
     pages = PAGES,
-    format = Documenter.HTML(;
-        prettyurls = get(ENV, "CI", "false") == "true",
-        canonical = "https://qutip.github.io/QuantumToolbox.jl",
-        edit_link = "main",
-        assets = ["assets/favicon.ico"],
-        mathengine = MathEngine,
-        size_threshold_ignore = ["api.md"],
+    # format = Documenter.HTML(;
+    #     prettyurls = get(ENV, "CI", "false") == "true",
+    #     canonical = "https://qutip.github.io/QuantumToolbox.jl",
+    #     edit_link = "main",
+    #     assets = ["assets/favicon.ico"],
+    #     mathengine = MathEngine,
+    #     size_threshold_ignore = ["api.md"],
+    # ),
+    format = DocumenterVitepress.MarkdownVitepress(
+        repo = "https://qutip.github.io/QuantumToolbox.jl",
+        # deploy_url = "https://qutip.org/QuantumToolbox.jl/",
```

Comment:
I think we need to uncomment this one

Reply:
Yes, I'm unsure. But using Documenter.jl, we didn't specify the deploy_url, everything was pushing to GitHub, like here. So, I think that the link is done in some other place